Resources/Private/Partials/Frontend/ContentElement/Service.html

```diff
@@ -0,0 +1,5 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true"></html>
```

Comment:
Das schließende Tag ist hier zu viel.
```suggestion
<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
```